scripts/build/rollup_emit_module_package_file.js

```diff
@@ -26,12 +26,14 @@
  */
 export function emitModulePackageFile() {
   return {
-    generateBundle() {
+    generateBundle(options) {
       this.emitFile({
         fileName: 'package.json',
         source: `{"type":"module"}`,
         type: 'asset'
       });
+      // I'm not sure why, this seems to prevent a rollup race condition?
+      console.log('Emitted module package.json file for:', options.file);
```

Comment:
Probably because console logs are relatively slow compared to everything else..?

Reply:
What is the race condition here?

Reply:
I think it might be gone - removed.